jwst/assign_wcs/nirspec.py

```diff
@@ -1228,31 +1229,58 @@ def compute_bounding_box(slit2detector, wavelength_range, slit_ymin=-.55, slit_y
 
     Parameters
     ----------
-    slit2detector : `astropy.modeling.core.Model`
-        The transform from slit to detector.
+    transform : `astropy.modeling.core.Model`
+        The transform from slit to detector, or detector to slit
```

Comment:
```suggestion
        The transform from slit to detector, or detector to slit.
        `nrs_wcs_set_input` uses "detector to slit", validate_open_slits uses "slit to detector".
```

Reply:
I suggest adding a comment to clarify which transform is used when.

Reply:
And `validate_open_slits` should be reworked in the future to have a forward and inverse transforms so that "compute_bounding_box" works consistently inthe two cases.